code/game/objects/items/weapons/nt_melee.dm

```diff
@@ -96,19 +136,112 @@
 		icon_state = initial(icon_state)
 	..()
 
+/obj/item/weapon/tool/sword/nt/scourge/apply_hit_effect(mob/living/carbon/human/target, mob/living/user, hit_zone)
+	. = ..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/O = target
+		target.stun_effect_act(stun, agony, hit_zone, src)
+		O.say(pick("OH", "LORD", "MERCY", "SPARE", "ME", "HAVE", "PLEASE"))
+
+/obj/item/weapon/tool/sword/nt/spear
+	name = "NT Pilum"
+	desc = "A saint looking short spear, designed for use with a shield or as a throwing weapon. The spear-tip usually breaks after being thrown at a target, but it can be welded into shape again."
+	icon_state = "nt_spear"
+	item_state = "nt_spear"
+	wielded_icon = "nt_spear_wielded"
+	force = 24
+	force_wielded_multiplier = 1.08
+	var/tipbroken = FALSE
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = SLOT_BACK | SLOT_BELT 
+	throwforce = 75 
+	armor_penetration = ARMOR_PEN_HALF
+	throw_speed = 3
+	price_tag = 150
+	matter = list(MATERIAL_BIOMATTER = 10, MATERIAL_STEEL = 5) // easy to mass-produce and arm the faithful
+
+/obj/item/weapon/tool/sword/nt/spear/equipped(mob/living/W)
+	. = ..()
+	if(is_held() && is_neotheology_disciple(W))
+		embed_mult = 0.1
+	else
+		embed_mult = initial(embed_mult)
+	if(isBroken)
+		force = WEAPON_FORCE_NORMAL
+		throwforce = WEAPON_FORCE_HARMLESS
+
+/obj/item/weapon/tool/sword/nt/spear/dropped(mob/living/W)
+	embed_mult = 300
+	..()
+
+/obj/item/weapon/tool/sword/nt/spear/on_embed(mob/user)
```

Comment:
this would remove the requirement for it to embed.
```suggestion
/obj/item/weapon/tool/sword/nt/spear/throw_impact(atom/hit_atom)
```

Reply:
don't think this is necessary, because it would also break when you throw it hits nothing but the floor, and throwing things is how many people organize items and hand things to each other, so this would be a nuisance to handle.

Reply:
implemented in a different way, it will now break when it hits a mob or an object, but not when it hits a turf.